README.md

```diff
@@ -17,16 +17,14 @@ If you want to know more you can head over to our website: https://sabnzbd.org.
 
 SABnzbd has a good deal of dependencies you'll need before you can get running. If you've previously run SABnzbd from one of the various Linux packages, then you likely already have all the needed dependencies. If not, here's what you're looking for:
 
-- `python` (SABnzbd version 2: only python 2.7.x. SABnzbd version 3: python 3.5.x and higher, often called python3)
-- `python-cheetah`
+- `python` (Python 3.5 and higher, often called python3)
 - `par2` (Multi-threaded par2 installation guide can be found [here](https://sabnzbd.org/wiki/installation/multicore-par2))
```

Comment:
Info on the required python modules is missing. Rather than list any modules in README.md I suggest adding a reference to requirements.txt, to avoid keeping this info in multiple places where it inevitably gets out of sync. In that file, the modules can be sorted into 'required' and 'optional/nice to have' categories with short comments on why where appropriate.

Reply:
True! Listed them and added a reference for `pygobject` with note to check `requirements.txt` for more info.